tests/core_widgets/test_snap_button_widget.py

```diff
@@ -0,0 +1,25 @@
+from micromanager_gui._core_widgets._snap_button_widget import SnapButton
+
+
+def test_snap_button_widget(qtbot, global_mmcore):
```

Comment:
Add the typing now so it's nicer to develop on in the future.

```suggestion
from __future__ import annotations
from micromanager_gui._core_widgets._snap_button_widget import SnapButton

from typing import TYPE_CHECKING
if TYPE_CHECKING:
    from pytestqt.qtbot import QtBot
    from pymmcore_plus import CMMCorePlus

def test_snap_button_widget(qtbot: QtBot, global_mmcore: CMMCorePlus):
```